test/SlowTests/Bugs/Distinct.cs

```diff
@@ -82,7 +82,6 @@ public void CanQueryForDistinctItemsUsingLinq(Options options)
                     var objects = s.Query<User>("test")
                         .Customize(x => x.WaitForNonStaleResults())
                         .Select(o => new { o.Name }).OrderBy(o => o.Name)
-                        .Select(o => new { o.Name })
```

Comment:
Why remove that? What is the behavior when this happens (users may have this)

Reply:
https://github.com/ravendb/ravendb/pull/15203

This will call `VisitSelect(,...)` in `RavenQueryProviderProcessor` twice. Since we wanted to disable chained selects this triggers it too.